src/coreclr/debug/ee/debugger.cpp

```diff
@@ -2774,6 +2774,13 @@ DebuggerMethodInfo *Debugger::GetOrCreateMethodInfo(Module *pModule, mdMethodDef
     }
     CONTRACTL_END;
 
+#ifdef DACCESS_COMPILE
+    if (!HasLazyData())
```

Comment:
I am not sure I understand how this change is related to fixing the problem

Reply:
Sure, I just wanted to make sure the change was intentional.

Reply:
Can you approve this PRm